packages/bundler-plugin-core/src/sentry/telemetry.ts

```diff
@@ -70,7 +70,7 @@ export function setTelemetryDataOnHub(options: NormalizedOptions, hub: Hub, bund
     );
   }
 
-  hub.setTag("module-metadata", !!options._experiments.moduleMetadata);
+  hub.setTag("module-metadata", !!options._experiments.moduleMetadata || !!options.moduleMetadata);
```

Comment:
Are we setting this tag before we normalize options? If not, we could remove the _experimental flag, right?

Reply:
oh you're right. This is after normalization. 👌